docusaurus.config.js

```diff
@@ -181,11 +182,11 @@ const config = {
             items: [
```

Comment:
Do we want to have a link to botkube.io site? Maybe clicking a logo should redirect to it? WDYT?
BTW If you agree, it can be done in a follow-up, actually that will be easier to not introduce the change in the same PR.

Reply:
We agreed that we don't want to do that. Marketing page opens docs page in the new tab so if someone wants to back to marketing page he/she can do it by changing a tab in the browser.

Reply:
Alright 👍